src/App.tsx

```diff
@@ -15,47 +16,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((good1, good2) => {
```

Comment:
good1 and good2 it's not good naming, you can rename them to firstGood and secondGood
or you can use just default name for sort params, like .sort((a, b) => {